src/main/java/simpaths/data/statistics/EmploymentStatistics.java

```diff
@@ -0,0 +1,70 @@
+package simpaths.data.statistics;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Id;
+import jakarta.persistence.Entity;
+
+import microsim.data.db.PanelEntityKey;
+import microsim.statistics.CrossSection;
+import microsim.statistics.IDoubleSource;
+import microsim.statistics.functions.MeanArrayFunction;
+import simpaths.data.filters.EmploymentHistoryFilter;
+import simpaths.model.SimPathsModel;
+import simpaths.model.enums.Les_c4;
+import simpaths.model.Person;
+
+@Entity
+public class EmploymentStatistics {
+
+    @Id
+    private PanelEntityKey key = new PanelEntityKey(1L);
+
+    @Column(name= "EmpToNotEmp")
+    private double EmpToNotEmp;         // Proportion of employed people becoming unemployed
+
+    @Column(name= "NotEmpToEmp")
+    private double NotEmpToEmp;         // Proportion of unemployed people becoming employed
+
+
+    public double getEmpToNotEmp() {
+        return EmpToNotEmp;
+    }
+
+    public void setEmpToNotEmp(double empToNotEmp) {
+        EmpToNotEmp = empToNotEmp;
+    }
+
+    public double getNotEmpToEmp() {
+        return NotEmpToEmp;
+    }
+
+    public void setNotEmpToEmp(double notEmpToEmp) {
+        NotEmpToEmp = notEmpToEmp;
+    }
+
+    public void update(SimPathsModel model) {
+
+        EmploymentHistoryFilter employmentHistoryEmployed = new EmploymentHistoryFilter(Les_c4.EmployedOrSelfEmployed);
+        EmploymentHistoryFilter employmentHistoryUnemployed = new EmploymentHistoryFilter(Les_c4.NotEmployed);
+
+
+        // Entering employment prevalence
+        CrossSection.Integer personsNotEmpToEmp = new CrossSection.Integer(model.getPersons(), Person.class, "getEmployed", true);
+        personsNotEmpToEmp.setFilter(employmentHistoryUnemployed);
+        // Entering unemployment prevalence
```

Comment:
Would this comment be more accurate if it referred to "Entering not employed transition rate"? Likewise the previous comment where prevalence could be changed to transition rate.

It may also be prudent not to refer to "unemployment" in the code since until the double hurdle model is introduced the "Les_c4.NotEmployed" category is a mixture of unemployed and some economic inactivity. Therefore, not strictly unemployed.

Otherwise, the statistics seem appropriate.

Reply:
Sounds good, will edit both:

```suggestion
        // Entering not employed transition rate
```